apps/dashboard/src/app/(internal)/event/components/event-filters.tsx

```diff
@@ -18,8 +19,19 @@ export const EventFilters = ({ onChange }: Props) => {
   }, [onChange, debouncedData])
 
   return (
-    <form>
-      <TextInput placeholder="Søk etter arrangementer..." {...form.register("bySearchTerm")} />
+    <form
+      onSubmit={(e) => {
+        e.preventDefault()
+      }}
+    >
+      <Group gap={4}>
+        <TextInput placeholder="Søk etter arrangementer..." {...form.register("bySearchTerm")} />
+        {Boolean(data.bySearchTerm) && (
+          <ActionIcon size="input-sm" variant="subtle" color="gray" type="reset" onClick={() => form.reset()}>
```

Comment:
Should probably be single field reset and not the whole form? Make it correct now so we don't have to fix it when we add more filters